cmd/rbac/main.go

```diff
@@ -1,7 +1,19 @@
 package main
 
-import "fmt"
+import (
+	"net/http"
+
+	routers "github.com/omc-college/management-system/pkg/rbac/api/routers"
+	postgres "github.com/omc-college/management-system/pkg/rbac/postgres"
+)
 
 func main() {
-	fmt.Println("I'm rbac service")
+	// Open DB
+	db, err := postgres.NewDB("postgres://postgres:superuser@localhost:5432/roles?sslmode=disable")
+	if err != nil {
+		panic(err.Error())
```

Comment:
let's use logrus package for logging `github.com/sirupsen/logrus` and instead of panic just use `log.Fatalf()`